website/content/en/preview/tasks/scheduling.md

```diff
@@ -176,12 +195,10 @@ metadata:
   name: gpu
 spec:
   requirements:
-  - key: node.kubernetes.io/instance-type
+  - key: karpenter.amazonaws.com/instance.family
```

Comment:
Using `amazonaws.com` is a bit awkward to me. We chose `k8s.aws` originally since that's what ACK used and it's short :) . Small nit is that amazonaws.com isn't necessarily the accurate DNS name either in China or some other regions, although I realize it doesn't necessarily need to map 1:1.   

Another argument to made here is that we're sub-dividing `amazonaws.com` with a karpenter subdomain for a general concept. Should this be more general so that other systems could use these same labels?  From a cloud-provider model perspective, it might make more sense to make karpenter the high-level domain and subdomain the CPs. 

`karpenter.sh/amazonaws/instance/family`
or
`karpenter.sh/aws/instance/family`
or
`k8s.aws/instance/family`

Reply:
Often this domain points to the docs site (e.g. karpenter.sh, knative.dev). I feel fairly strongly that it should be a valid DNS name, but agree that amazonaws.com is both verbose and seemingly a bit inaccurate. Google uses cloud.google.com, which seems like a single, simple, irrefutable domain :P

Reply:
Discussed offline -- keeping karpenter.k8s.aws